data_classes/lib/data_classes.dart

```diff
@@ -2,10 +2,13 @@ import 'package:collection/collection.dart';
 import 'package:meta/meta.dart';
 
 // export 'dart:async';
-export 'package:collection/collection.dart' hide IterableExtension;
-export 'package:json_annotation/json_annotation.dart';
+export 'package:collection/collection.dart'
+    hide IterableExtension, IterableNullableExtension;
+export 'package:dartx/dartx.dart' show IterableMapNotNull;
```

Comment:
Why to export dartx from here?

Reply:
I only export the mapNotNull function of dartX because I use it on the deserialization code in the generated data classes.